tests/test_datasets/test_internal_datasets.py

```diff
@@ -239,12 +241,13 @@ def test_not_present_part():
             pd.to_datetime("2018-06-26 19:00:00"),
             ("train", "test"),
         ),
-        (
+        pytest.param(
```

Comment:
Is this the only dataset we have troubles with?

Reply:
I remove `m4_daily`, `m4_monthly` and `IHEPC_T`